tests/modules/services/copyq/basic-configuration.nix

```
@@ -0,0 +1,15 @@
+{ ... }:
+
+{
+  config = {
+    services.copyq = {
+      enable = true;
+      systemdTarget = "sway-session.target";
+    };
```

It's best to avoid pulling in the actual copyq package as a dependency since it'll cause unnecessary downloads. Instead can try stubbing the package:

```suggestion
    };
    
    test.stubs.copyq = { };
```